codec/protobuf/protobuf_test.go

```diff
@@ -0,0 +1,19 @@
+package protobuf
+
+import (
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/asdine/storm/codec/internal"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestProtobuf(t *testing.T) {
+	u1 := SimpleUser{
+		Id: proto.Uint64(1),
+		Name: proto.String("John"),
+	}
+	u2 := SimpleUser{}
+	internal.RoundtripTester(t, Codec, &u1, &u2)
+	assert.True(t, u2.GetId() == u2.GetId())
```

Comment:
> u1 and u2
 
thanks to notice it:

https://github.com/bclermont/storm/commit/9624229e3a2afb65002d521eded04df3e46fde63

```
go test -v
=== RUN   TestProtobuf
--- PASS: TestProtobuf (0.00s)
PASS
ok  vendor/github.com/asdine/storm/codec/protobuf	0.010s
```